app/views/shared/_available_languages.html.erb

```diff
@@ -0,0 +1,18 @@
+<div class="available-languages">
```

Comment:
Will this be needed? this is copied from `verify-frontend`. Allows you to switch between languages

Reply:
no, it's not needed